blocks/components/editable/index.js

```diff
@@ -127,6 +146,22 @@ export default class Editable extends wp.element.Component {
 		}
 	}
 
+	componentWillReceiveProps( nextProps ) {
+		forEach( nextProps.formats, ( state, format ) => {
+			const currentState = this.formats[ format ] || false;
```

Comment:
Is `this.formats` here any different than `this.props.formats`? Like Riad, I also instinctively cringe at the sight of instance variables, though they're sometimes justified.